pypi/pip_generate.go

```diff
@@ -509,7 +513,7 @@ func main() {
 
 		sort.Sort(wheelsByPlatform(wheels))
 		dependencies = append(dependencies, pyPIDependency{packageName, wheels})
-		installedPackages[packageName] = true
+		installedPackages[normalizePackageName(packageName)] = true
```

Comment:
Ah, yep, I see the problem: this comes from the wheel file name which may have uppercase characters, while the dependency name is normalized so it will not. Nice.